src/test/scala/uk/gov/nationalarchives/FileProcessorTest.scala

```diff
@@ -330,10 +330,15 @@ class FileProcessorTest extends AnyFlatSpec with MockitoSugar with TableDrivenPr
     forAll(citeTable) { (potentialCite, idFields) =>
       forAll(treNameTable) { (treName, treFileName, expectedFolderTitle, expectedAssetTitle) =>
         forAll(urlDepartmentAndSeriesTable) { (department, series, _, parsedUri, expectedFolderName, titleExpected) =>
-          val updatedIdFields =
-            if (potentialCite.isDefined && expectedFolderName == trimmedUri) idFields :+ IdField("URI", trimmedUri)
-            else idFields
-          "createBagitMetadataObjects" should s"generate the correct bagit Metadata with $potentialFileReference $expectedFolderTitle, $expectedAssetTitle and $updatedIdFields" +
+          val updatedIdFields = {
```

Comment:
1. You don't really need this outer brace (the one that wraps the if expression)
2. Since the if and else are one line, do you think it's worth removing the braces or do you think that would make it unreadable?
```  
        val updatedIdFields =
            if (department.isEmpty) Nil
            else
              if (potentialCite.isDefined && expectedFolderName == trimmedUri) idFields :+ IdField("URI", trimmedUri)
              else idFields```

Reply:
Good point on both, I've removed the braces and change the if as you suggested.